typescript/react-navigation.d.ts

```diff
@@ -1467,4 +1452,16 @@ declare module 'react-navigation' {
   export const NavigationContext: React.Context<NavigationScreenProp<NavigationRoute>>;
   export const StackGestureContext: React.Context<React.Ref<PanGestureHandler>>;
   export const DrawerGestureContext: React.Context<React.Ref<PanGestureHandler>>;
+  
+  /**
+   * SceneView
+   */
+
+  export interface SceneViewProps {
+    component: React.ComponentType;
+    screenProps: ScreenProps;
+    navigation: NavigationScreenProp;
```

Comment:
looks like this is causing an error on CI: https://circleci.com/gh/react-navigation/react-navigation/4289#tests/containers/1

Reply:
It should be resolved by now